### PR TITLE
RB-form: #1 - form for work experience

### PR DESCRIPTION
**Pull request by Esha-Rajbhandari**
### RB-form: #1 - form for adding resume

Capture

Capture1

### CODE REVIEW

File: src/components/inputtextarea/InputTextArea.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InputTextArea = ({label, placeholder}) => {
+  return(
+    <div className="input-group">
```
Comment (elizaRjb) [inline src/components/inputtextarea/InputTextArea.js:5]: This class can be named as `input__group`

File: src/components/inputtextarea/InputTextArea.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      <label className="input-label">{label}</label>
```
Comment (elizaRjb) [inline src/components/inputtextarea/InputTextArea.js:6]: This can be named as `input__label`

File: src/components/inputtextarea/InputTextArea.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      <textarea className="input-textarea-field" cols="30" rows="10" placeholder={placeholder}></textarea>
```
Comment (elizaRjb) [inline src/components/inputtextarea/InputTextArea.js:7]: this can be named as `input__textarea`

File: src/components/inputtext/InputText.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InputText = ({label, placeholder}) => {
+  return(
+    <div className="input-group">
+      <label className="input-label">{label}</label>
+      <input type="text" className="input-text-field" placeholder={placeholder}/>
```
Comment (elizaRjb) [inline src/components/inputtext/InputText.js:7]: this can be named as `input__text-field`

File: src/components/inputselect/InputSelect.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const InputSelect = ({label}) => {
+  return(
+    <div className="input-group">
+      <label className="input-label">{label}</label>
+      <select name="" id="" className="input-select-field">
```
Comment (elizaRjb) [inline src/components/inputselect/InputSelect.js:7]: this can be named as `input__select`

Comment (elizaRjb) [reply src/components/inputselect/InputSelect.js:7]: Use BEM format where necessary. For example, all these form elements fall under FORM block, or in this case INPUT block. And `input-label` and `input-group` typically does not exist outside form.

File: src/components/inputtextarea/InputTextArea.js (hunk 1 above)
Comment (Esha-Rajbhandari) [reply src/components/inputtextarea/InputTextArea.js]: 🆗

File: src/components/inputtextarea/InputTextArea.js (hunk 2 above)
Comment (Esha-Rajbhandari) [reply src/components/inputtextarea/InputTextArea.js]: 🆗

File: src/components/inputtextarea/InputTextArea.js (hunk 3 above)
Comment (Esha-Rajbhandari) [reply src/components/inputtextarea/InputTextArea.js]: 🆗

File: src/components/inputtext/InputText.js (hunk 4 above)
Comment (Esha-Rajbhandari) [reply src/components/inputtext/InputText.js]: 🆗

File: src/components/inputselect/InputSelect.js (hunk 5 above)
Comment (Esha-Rajbhandari) [reply src/components/inputselect/InputSelect.js]: 🆗

File: src/assets/sass/modules/Form/_form.scss [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+  @include e(radio-field) {
+    margin-top: 18px;
+  }
+
+  @include e(radio-field-content) {
+    display: flex;
+    align-items: center;
+  }
+
+  .input {
```
Comment (elizaRjb) [inline src/assets/sass/modules/Form/_form.scss:34]: Keep this `input` outside of `form`. Keeping it inside will only create unnecessary nesting and make it more complex. you can create a separate `_input.scss` partial

File: src/components/dashboard/Dashboard.js [hunk 7]
```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import Skills from '~/components/skills/Skills';
 import Sidenav from '~/components/sidenav/Sidenav';
+import AddSkill from '~/components/form/skill/AddSkill';
+import AddProject from '~/components/form/project/AddProject';
 import Certificate from '~/components/certificate/Certificate';
 import Achievements from '~/components/achievements/Achievements';
 import WorkExperience from '~/components/workexperience/WorkExperience';
+import AddContactInformation from '~/components/form/contact/AddContact';
```
Comment (sagunji) [inline src/components/dashboard/Dashboard.js:9]: wow, so many components are being imported. Will it all be used in this component.

File: src/assets/sass/modules/Form/_form.scss (hunk 6 above)
Comment (Esha-Rajbhandari) [reply src/assets/sass/modules/Form/_form.scss]: 🆗

File: src/components/dashboard/Dashboard.js (hunk 7 above)
Comment (Esha-Rajbhandari) [reply src/components/dashboard/Dashboard.js:9]: The form component can be removed. Will make the changes.

Comment (Esha-Rajbhandari) [reply src/components/dashboard/Dashboard.js]: Fixed.

**Pull request merged**